storage/boltz/store_query.go

```diff
@@ -134,7 +134,7 @@ func (store *BaseStore) createCompositeEntitySymbol(name string, first linkedEnt
 			chain:      chain,
 		}
 	}
```

Comment:
Was assuming that if there was a non-set component to the chain that it would be last, but non-set components can be anywhere in the chain. This is just deciding how to extract the last value, so if the last value is a set, use the set appropriate method, otherwise use the single value appropriate method.